src/components/Dialogs/elements/MultiOption.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div>
+    <p class="mb-2 text-xs">
+      {{ modifiedItem === 'task' ? 'Subtasks' : 'Board Columns' }}
+    </p>
+
+    <div class="flex flex-col gap-3">
+      <div
+        v-for="item in selectedMultiOptionItems"
+        :key="item"
+        class="flex items-center gap-2"
+      >
+        <input-text
+          :placeholder="
+            modifiedItem === 'task' && formType === 'add' ? item : undefined
+          "
+          :customValue="
+            modifiedItem === 'task' && formType === 'add' ? undefined : item
+          "
+          :formType="formType"
+          type="text"
+          :name="item"
+          :error="isError"
+          :class="{ 'after:content-none': !isError }"
+          class="subtask-input"
+        />
+        <svg
+          width="15"
+          height="15"
+          xmlns="http://www.w3.org/2000/svg"
+          class="cursor-pointer"
```

Comment:
Because those are the closing icons:
![add_form](https://github.com/Kepmon/task-manager/assets/113309294/f755d463-500e-4650-8bbe-4397e08a0209)

But you're right, it should be a button along with the respective aria-label and some padding around it 😛 
**Corrected**